server/handlers/shared_validators.js

```diff
@@ -183,6 +183,12 @@ exports.walletBodyValidator = body("wallet")
   .bail()
   .isLength({ min: 0 });
 
+exports.hasPendingCancelationValidator = query("hasPendingCancelation")
+  .optional()
+  .notEmpty()
+  .bail()
+  .isBoolean();
+
```

Comment:
It should be only "true" or "false" based on what you wrote in the client handler